<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AuthRegDialog</class>
 <widget class="QDialog" name="AuthRegDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>487</width>
    <height>498</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <property name="leftMargin">
      <number>120</number>
     </property>
     <property name="topMargin">
      <number>24</number>
     </property>
     <property name="rightMargin">
      <number>120</number>
     </property>
     <property name="bottomMargin">
      <number>24</number>
     </property>
     <item>
      <layout class="QVBoxLayout" name="verticalLayoutA">
       <item>
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="label">
         <property name="maximumSize">
          <size>
           <width>999</width>
           <height>32</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>JetBrains Mono</family>
           <pointsize>18</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Авторизация</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>24</height>
          </size>
         </property>
         <property name="text">
          <string>Логин:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit">
         <property name="placeholderText">
          <string>введите логин</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>24</height>
          </size>
         </property>
         <property name="text">
          <string>Пароль:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_2">
         <property name="placeholderText">
          <string>введите пароль</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_3">
         <property name="text">
          <string>Войти</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayoutR">
       <item>
        <spacer name="verticalSpacer_7">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="regHeading">
         <property name="maximumSize">
          <size>
           <width>999</width>
           <height>32</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>JetBrains Mono</family>
           <pointsize>18</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Регистрация</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_8">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="regLabel1">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>24</height>
          </size>
         </property>
         <property name="text">
          <string>Логин:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="regLogin">
         <property name="placeholderText">
          <string>введите логин</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="regLabel2">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>24</height>
          </size>
         </property>
         <property name="text">
          <string>Пароль:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="regPass1">
         <property name="placeholderText">
          <string>введите пароль</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_9">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="regBtn">
         <property name="text">
          <string>Зарегистрироваться</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
